t/integration/test_inspect.py

```diff
@@ -47,10 +48,17 @@ def test_clock(self, inspect):
     @flaky
     def test_registered(self, inspect):
         """Tests listing registered tasks"""
+        # TODO: We can check also the exact values of the registered methods
```

Comment:
we can also check the exact values of......

Reply:
Yes we can but it is tricky. To do it properly we need load separate set of tasks in this test and this needs more work in loading worker which I wanted to avoid right now...